app/styles/404.css

```diff
@@ -0,0 +1,12 @@
+.not-found-container{
+  margin: auto;
+  text-align: center;
+}
+
+.not-found-img {
+  width: 60%;
+}
+
+.not-found-text{
+  font-weight: bold;
+}
```

Comment:
For the mobile devices, can we please make the font size a bit small somewhere around `18px` ?

Reply:
> Image with media query added for small devices
![image](https://user-images.githubusercontent.com/85296770/209690026-61c29154-1ea7-483a-8c58-1c39dd053a76.png)